Fix #23, Fix #27

redburningice opened the pull request:
Added driver adjustable settings to lap_data

redburningice committed:
Fix #23, Fix #27

Added driver adjustable settings to lap_data

redburningice committed:
Re-add project file

msundstrom commented on ACCStatsUploader/Converters/CarModel.cs:
```diff
@@ -0,0 +1,103 @@
+
+namespace ACCStatsUploader.Converters {
+    
+public class CarModel {
```
Formatting thing: this block should be indented 1 tab.

msundstrom commented on .gitignore:
```diff
@@ -3,4 +3,6 @@ ACCStatsUploader/obj/*
\ No newline at end of file
+obj/
+testingPlayground
+ACCStatsUploader.sln
```
This should not be in the `.gitignore`.

redburningice commented on ACCStatsUploader/Converters/CarModel.cs:
```diff
@@ -0,0 +1,103 @@
+namespace ACCStatsUploader.Converters {
+    
+public class CarModel {
+        public enum KUNOS_ID {
```
rename KUNOS_ID to car_model

redburningice commented on ACCStatsUploader/Converters/BrakeBalanceConverter.cs:
```diff
@@ -0,0 +1,61 @@
+
+namespace ACCStatsUploader.Converters {
+    public class BrakeBalanceConverter {
+        private static Dictionary<CarModel.KUNOS_ID, int> brakeBalanceOffset = new Dictionary<CarModel.KUNOS_ID, int> {
```
replace this dictionary with an extension method for the enumeration KUNOS_ID.
https://stackoverflow.com/a/5985710

redburningice committed:
Implement Mikael's suggestions, remove false entries from .gitignore

Write ElectronicsInfo.getGroupedListOutput more compactly with String.Join. Replace the CarModel and BrakeBalanceConverter dictionaries with an extension method for the enumeration KUNOS_ID

redburningice committed:
remove entries from gitignore file

msundstrom commented on ACCStatsUploader/Sheets/LapSheet.cs:
```diff
@@ -162,6 +167,11 @@ public class LapSheet : Sheet {
                 new Cell { value = lapInfo.tyreInfo.maxPressures().fr },
                 new Cell { value = lapInfo.tyreInfo.maxPressures().rl },
                 new Cell { value = lapInfo.tyreInfo.maxPressures().rr },
+                new Cell { value = lapInfo.electronicsInfo.listTC()},
+                new Cell { value = lapInfo.electronicsInfo.listTC2()},
+                new Cell { value = lapInfo.electronicsInfo.listABS()},
+                new Cell { value = lapInfo.electronicsInfo.averageBrakeBalance()},
+                new Cell { value = lapInfo.electronicsInfo.listFuelMap()},
```
```suggestion
                new Cell { value = lapInfo.electronicsInfo.listTC() },
                new Cell { value = lapInfo.electronicsInfo.listTC2()},
                new Cell { value = lapInfo.electronicsInfo.listABS() },
                new Cell { value = lapInfo.electronicsInfo.averageBrakeBalance() },
                new Cell { value = lapInfo.electronicsInfo.listFuelMap() },
```

msundstrom reviewed (approved):
Looks good, let's get this merged!
